runtime/darwinia/src/lib.rs

```diff
@@ -563,6 +573,80 @@ sp_api::impl_runtime_apis! {
 		}
 	}
 
+	impl darwinia_fee_market_rpc_runtime_api::FeeMarketApi<Block, Balance> for Runtime {
+		fn market_fee() -> Option<Fee<Balance>> {
+			if let Some(fee) = FeeMarket::market_fee() {
+				return Some(Fee {
+					amount: fee,
+				});
+			}
+			None
+		}
+
+		fn in_process_orders() -> InProcessOrders {
+			return InProcessOrders {
+				orders: FeeMarket::in_process_orders(),
+			}
+		}
+	}
+
+	impl bridge_primitives::CrabFinalityApi<Block> for Runtime {
+		fn best_finalized() -> (BlockNumber, Hash) {
+			let header = BridgeCrabGrandpa::best_finalized();
+			(header.number, header.hash())
+		}
+
+		fn is_known_header(hash: Hash) -> bool {
+			BridgeCrabGrandpa::is_known_header(hash)
+		}
+	}
+
+	impl bridge_primitives::ToCrabOutboundLaneApi<Block, Balance, crab_messages::ToCrabMessagePayload> for Runtime {
+		fn estimate_message_delivery_and_dispatch_fee(
+			_lane_id: bp_messages::LaneId,
+			payload: crab_messages::ToCrabMessagePayload,
+		) -> Option<Balance> {
+			bridge_runtime_common::messages::source::estimate_message_dispatch_and_delivery_fee::<crab_messages::WithCrabMessageBridge>(
+				&payload,
+				crab_messages::WithCrabMessageBridge::RELAYER_FEE_PERCENT,
+			).ok()
+		}
```

Comment:
can be comment out.

Reply:
In bridger, Plan to stop conversion
https://github.com/darwinia-network/bridger/pull/321